book/website/collaboration/inclusive-events/inclusive-events-content-guide.md

```diff
@@ -0,0 +1,95 @@
+
+This can mean ensuring that any video content is accompanied by captions, providing Alternative Text descriptions for images, and that any controls or clickable features on the site have text labels that describe the purpose of the feature.
+ For example, a button labelled "Sign Up to Newsletter" or "Upload Image". 
+
+### 2. **Operability**
+Operability means that all users can control and interact with any buttons or clickable features.
+
+To meet this requirement, ensure that any buttons are able to be interacted with and clicked through different modalities for example, a mouse, a keyboard, or voice command.
+ 
+This is especially important for registration forms which often impose a time limit to complete them. 
+Any registration forms you use for your event must have a way to extend - or turn off - the time limit function for completing the form.
+
+If you are using an external registration website, please check the accessibility guidance of that site to ensure that this feature is enabled.
+It is incredibly important for people who use screen readers and others who may need more time to read and complete a form due to dyslexia, familiarity with the event's host language, or any other reason that impacts reading and language comprehension.
+
+Operability also requires that web pages have an informative title for the content on the page.
+
+### 3. **Understandability** 
+Understandability refers to how easy is it for a general audience to understand your website's use and content.
+
+For example, content being written in easy read, or plain language formats. This is discussed in more depth in the {ref}`cl-inclusive-events-comms-plain-language` section below.
+
+The Web Accessibility guidelines ask that any technical terms, jargon, idioms, slang and abbreviations be accompanied by a definition.
+
+Ideally, your site will also contain references to a Glossary that is linked to when those words appear.
+You may need to develop a Glossary specific to your event and area, or consider what Glossaries have been developed by research funding bodies or University Communications teams and provide links if applicable. 
+
+### 4. **Robustness**.
+Robustness refers to the importance of content on your event page's website being compatible with different browsers, now and in the future, and how compatible the content is with different technologies such as screen readers and other assistive technologies. 
+
+## Conclusion
+The guidance provided here is not an exhaustive list of accessibility requirments, and with all other planning of your event, it is best to work with the specific group and demographic of attendees that will be at your event. 
+This can help ensure that any content accessibility requirements specific to your attendees can be identified and addressed before the start of your event.
+  
+  
```

Comment:
```suggestion
  **Resources:**
  Below, we share some resources so that you can find out about events and accessible material.

-  [Buenas prácticas para crear presentaciones accesibles](https://www.metadocencia.org/post/presentaciones-accesibles/)

- [Checklist para presentaciones accesibles](https://drive.google.com/file/d/1sBeuVn8koBm9vRkcyTvW5yEN6F109lVs/view) 

- [Cómo hacer reuniones virtuales pensadas para todas las personas](https://www.metadocencia.org/post/accesibilidad_1/)

- [How to Make Virtual Meetings Accessible](https://www.levelaccess.com/blog/how-to-make-virtual-meetings-accessible)

- [Make your PowerPoint presentations accessible to people with disabilities](https://support.microsoft.com/en-us/office/make-your-powerpoint-presentations-accessible-to-people-with-disabilities-6f7772b2-2f33-4bd2-8ca7-dae3b2b3ef25#:~:text=In%20PowerPoint%2C%20the%20Accessibility%20Checker,%2C%20select%20Review%20%3E%20Check%20Accessibility)

```